modules/social_features/social_event/config/static/core.entity_view_display.node.event.email_card_11100.yml

```diff
@@ -0,0 +1,68 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.email_card
+    - field.field.node.event.field_event_address
+    - field.field.node.event.field_event_date
+    - field.field.node.event.field_event_location
+    - node.type.event
+  module:
+    - address
+    - datetime
+    - user
+  enforced:
+    module:
+      - social_event
+id: node.event.email_card
+targetEntityType: node
+bundle: event
+mode: email_card
+content:
+  field_event_address:
+    type: address_plain
+    weight: 1
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    region: content
+  field_event_addtocal:
```

Comment:
It's an optional submodule, so not sure if this field_event_addtocal should be in this config.

Reply:
@rollki I removed this field from view display. Even more, if we take a look to [this function](https://github.com/goalgorilla/open_social/blob/main/modules/social_features/social_event/modules/social_event_addtocal/social_event_addtocal.module#L32) we can see that the field `field_event_addtocal` can't be managed in any view displays 🤷